mobile/library/kotlin/io/envoyproxy/envoymobile/grpc/GRPCStream.kt

```diff
@@ -51,9 +51,19 @@ class GRPCStream(
   }
 
   /**
-   * Close this connection.
+   * Close the stream, wait for the peer to finish before actually closing the stream.
```

Comment:
Are there cases where this is used?  I thought the plan was to replace it entirely

Reply:
I do not think that we will need it at Lyft so I am fine with removing it but since I want to run an A/B experiment first I want to keep `close` method for the time being so that my control branch continues to do what it does now.

Going to follow up with the removal if we find out that `cancel` is a clear win for us.

Reply:
I can add a TODO.

Reply:
If you decide to keep this method (and I'm happy for it to go away :>) I wonder if we might want to say something likes, "Close these stream by sending the "end stream" signal to the peer and then wait for the peer to finish before the stream will actually close"

But less code is better if we can get away with it!